client/src/components/Listings.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import Modal from './Modal.jsx';
+import LoveHeart from './LoveHeart.jsx';
+
+class Listings extends React.Component {
+  constructor(props){
+    super(props);
+
+    this.state = {
+      roomInfo: [],
+      heartClicked: false
+    }
+  }
+
+  clickHandler(){
+    console.log('CLIIIICK');
+  }
+
+  render(){
+
+
+    let roomData = this.props.roomInfo.map((data) => {
+      return(
+        <div className="img-container" style={{transform: `translateX(${this.props.transform}%)`}}>
```

Comment:
BRILLIANT! I'll be able to apply `transform` css property for part of my calendar. Thank you so much, Amar! You saved me from a lot of time researching.